Example/Tests/ILGClasses_Tests_ObjC.m

```diff
@@ -35,8 +41,17 @@ - (void)testNonIncludedTestWorks
                                                    ]];
     
     NSSet *retrievedClasses = [ILGClasses classesPassingTest:^BOOL(Class class) {
-        NSString *className = NSStringFromClass(class);
-        return [className containsString:@"Class1"];
+        if ([ILGClasses class:class orAnyOfItsSuperclassesPassesTest:^BOOL(Class  _Nonnull class) {
+            return class == [NSObject class];
+        }]) {
```

Comment:
Unfortunately, that looks goofy as hell with the `orAny...` being so long. How bout this: 

```objc
BOOL inheritsFromNSObject =  [ILGClasses class:class orAnyOfItsSuperclassesPassesTest:^BOOL(Class _Nonnull class) {
            return class == [NSObject class];
        }];
        
        if (inheritsFromNSObject) {
             ...
```